fragdenstaat_de/fds_blog/models.py

```diff
@@ -420,6 +420,14 @@ def get_authors(self):
     def get_authors_string(self):
         return ", ".join(str(author) for author in self.get_authors())
 
+    def get_authors_link_list(self):
+        links = []
+        for author in self.get_authors():
+            url = reverse("blog:article-author", kwargs={"username": author.username})
+            link_text = author.get_full_name()
+            links.append("<a href='{}'>{}</a>".format(url, link_text))
```

Comment:
Do we need to escape the `link_text` here to prevent XSS etc? Or is the name field already safe enough?